templates/identity/email-password/confirm.html.ejs

```diff
@@ -1,4 +1,16 @@
 <h1>Authorize</h1>
```

Comment:
Maybe we also want a <p> below this to explain what's happening.
Even if just

`<p>You are authorizing an application to access your Pod.</p>`

Reply:
We have #827 for fixing this more decently, but I can already add that line.